pkg/sql/pgwire/testdata/pgtest/tuple

```diff
@@ -235,22 +235,3 @@ ReadyForQuery
 {"Type":"ParseComplete"}
 {"Type":"ErrorResponse","Code":"42601","Message":"error in argument for $1: insufficient bytes reading element for binary format. elementIdx=0 bufferLength=12 bufferStartIdx=12 bufferEndIdx=13"}
 {"Type":"ReadyForQuery","TxStatus":"I"}
-
-# negative element size
```

Comment:
This would probably only fail with 32-bit go or a large tuple full of large items (not even sure if it is possible though because 2^32/2-1 < 2^64/2-1).